packages/@sanity/portable-text-editor/src/utils/patchToOperations.ts

```diff
@@ -45,11 +45,11 @@ export function createPatchToOperations(
             ? (parsed.start1 || 0) + parsed.diffs[0][1].length
             : (parsed.start2 || 0) + parsed.length2 - distance,
       }
-      debug(
-        `DiffMatchPatch (${distance < 0 ? 'remove' : 'insert'}) at ${JSON.stringify(slatePath)}}: `,
-        JSON.stringify(point, null, 2),
-        JSON.stringify(parsed, null, 2)
-      )
+      // debug(
```

Comment:
seems useful to have this enabled? (if put inside a function the function will only run if debug is enabled)